src/schema/v2/me/myCollectionCreateArtworkMutation.ts

```diff
@@ -122,12 +126,19 @@ export const myCollectionCreateArtworkMutation = mutationWithClientMutationId<
 
       const artworkId = response.id
 
-      if (editionNumber || editionSize) {
+      const providedEditionField = editionNumber || editionSize
```

Comment:
nitpick: `providedEditionField` sounds like you're going to use the content of the field; maybe `hasEditionField`? Or actually I think you could just skip the const and munge together the boolean without sacrificing readability.